doc/cloud/cloud-api-authenticator.rst

```diff
@@ -0,0 +1,258 @@
+Authenticator SDK
+'''''''''''''''''
+
+KNoT Cloud authenticator service JavaScript library
+
+----------------------------------------------------------------
+
+Quickstart
+----------
+
+Install
+#######
```

Comment:
The pattern used in this documentation subsection is marked with '.
Change # for '